src/components/pages/RoleSignup/Applications/README.md

```diff
@@ -0,0 +1,12 @@
+## Application and onboarding flow
+
+User will apply as a mentor/mentee (Mentor.js/Mentee.js).
+On submission of their application the user will be directed to '/apply/success' as confirmation that their application was successful.
+
+When/if the superAdmin approves the user, then the user will receive an email confirmation that they've been accepted.
+They will also receive a link to create their login information.
+
+It is unlikely, but possible, that the superAdmin will deny the application--in which case the user will receive an email to this effect.
+
+This flow is designed to save the stakeholder costs for creating users that MIGHT not continue with the program.
+This flow only creates a user after it is confirmed that the user will continue.
```

Comment:
Great README, I love this strategy to help explain the current state to devs without having to find a video or the person who made the changes.